hadoop-hdds/docs/content/feature/PrefixFSO.md

```diff
@@ -23,29 +23,33 @@ summary: Supports atomic rename and delete operation.
   limitations under the License.
 -->
 
-The prefix based FileSystem optimization feature supports atomic rename and
- delete of any directory at any level in the namespace. Also, it will perform
-  rename and delete of any directory in a deterministic/constant time.
+The prefix-based File System Optimization feature supports atomic rename and delete of any directory at any level in the
+namespace in deterministic/constant time.
 
-Note: This feature works only when `ozone.om.enable.filesystem.paths` is
- enabled which means that Hadoop Compatible File System compatibility is
-  favored instead of S3 compatibility. Some irregular S3 key names may be
-   rejected or normalized.
+This feature can be enabled for each specific bucket that requires it by setting the `--layout` flag
+to `FILE_SYSTEM_OPTIMIZED` at the time of bucket creation.
 
-This feature is strongly recommended to be turned ON when Ozone buckets are
- mainly used via Hadoop compatible interfaces, especially with high number of
-  files in deep directory hierarchy.
+```bash
+ozone sh bucket create /<volume-name>/<bucket-name> --layout FILE_SYSTEM_OPTIMIZED
+```
+
+Note: File System Optimization favors Hadoop Compatible File System over S3 compatibility. Some irregular S3 key names
+may be rejected or normalized.
+
+This feature is strongly recommended to be turned ON for Ozone buckets mainly used via Hadoop compatible interfaces,
+especially with high number of files in deep directory hierarchy.
+
+Previous versions of Ozone allowed cluster-wide prefix based
+file system optimization by enabling `ozone.om.enable.filesystem.paths` configuration.
 
 ## OzoneManager Metadata layout format
-OzoneManager supports two metadata layout formats - simple and prefix.
-
-Simple is the existing OM metadata format, which stores key entry with full path
- name. In Prefix based optimization, OM metadata format stores intermediate
-  directories into `DirectoryTable` and files into `FileTable` as shown in the
-   below picture. The key to the table is the name of a directory or a file
-    prefixed by the unique identifier of its parent directory, `<parent
-     unique-id>/<filename>`. 
-     
+OzoneManager supports two metadata bucket layout formats - Object Store and File System Optimized.
```

Comment:
Howe about including the acronym ? The idea is to popularise the terms eventually :-)

`- Object Store (OBS) and File System Optimized (FSO)`